interactions/api/models/guild.py

```diff
@@ -3036,6 +3036,37 @@ def discovery_splash_url(self) -> Optional[str]:
             else None
         )
 
+    @classmethod
+    async def get_all_guilds(cls, http: "HTTPClient") -> List["Guild"]:
```

Comment:
Maybe change it/set Union to bot? That might be an option to hide http (your idea)

Reply:
Good idea, I'd rather expose bot than http!